src/main/java/org/spongepowered/common/mixin/core/network/MixinNetHandlerPlayServer.java

```diff
@@ -95,6 +103,8 @@
 
     private Location lastMoveLocation = null;
 
+    private final Map<String, ResourcePack> sentResourcePacks = new HashMap<String, ResourcePack>();
```

Comment:
My preference is to have the field close to where it's used, especially for classes like NetHandlerPlayServer where there are lots of different handlers scattered about. That's why I put `tmpQuitMessage` where it is. Don't know if the rest of the Sponge team approve but that's my preference.